shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -1313,7 +1312,7 @@ \subsection{Reward Update Calculation}
   \begin{align*}
     & \fun{createRUpd} \in \BlocksMade \to \EpochState \to \Coin \to \RewardUpdate \\
     & \createRUpd{b}{es}{total} = \left(
-      \Delta t_1+\Delta t_2,-~\Delta r,~\var{rs},~-\var{feeSS}\right) \\
+      \Delta t,-~\Delta r,~\var{rs},~-\var{feeSS}\right) \\
```

Comment:
Since we've made it a minimal change for the moment, to make the change clear, we need to make the change in the spec be the same minimal change.

So it would only be moving `\Delta t_2` from the treasury side to the reserves side on this line.

Reply:
yes, the tex is in line with #1699, I will fix that now!

Reply:
I concur with @dcoutts . The TeX should reflect the code change as closely as possible.

Reply:
they are in sync now